src/relentless/simulate/lammps.py

```diff
@@ -212,7 +224,19 @@ def _call_commands(self, sim):
             sim.types, x=sim.potentials.pair.squared_space, tight=True, minimum_num=2
         )
         Nr = len(r)
-        sim[self]["_potentials_rmax"] = r[-1]
+
+        if has_bonds:
+            uB = collections.FixedKeyDict(sim[self]["bonds"].type_label.types)
+            fB = collections.FixedKeyDict(sim[self]["bonds"].type_label.types)
```

Comment:
Won't `sim[self]["bonds"]` only be available on the root rank? You probably need to store the bond type map in a plain dict and broadcast it, like is done for the particle types. The array of bonds themselves can be stored in a different key & only on the root rank (None on the other ranks) like for the HOOMD snapshot.

Reply:
I'm a bit confused on this portion. Would the bond type map be made above where the other portions are being broadcast?

Reply:
If you run this in MPI, does it work or does it throw an error that it can't find the `"bonds"` key? I think it will error out.

You need to make a dictionary from the type map while you have the snapshot acquired then broadcast it. You may additionally allow the user to specify a type map... take a look at how the `types` are handled currently, and you will need to do basically the same thing for the bonds, etc.